content/javascript/concepts/mutators/mutators.md

```diff
@@ -0,0 +1,101 @@
+---
+Title: 'Mutators'
+Description: 'Mutators change the value of private variables using the `set` keyword.'
+Subjects:
+  - 'Web Development'
+  - 'Computer Science'
+Tags:
+  - 'OOP'
+  - 'Methods'
+  - 'Objects'
+  - 'Classes'
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/front-end-engineer-career-path'
+---
+
+Developers can prevent direct modification of a **member variable** by creating a **private variable** and a **mutator** method (also known as a **setter**). This method determines the behavior when assigning a value to those variables.
```

Comment:
```suggestion
A **mutator** method (also known as a *setter*) can be used to prevent direct modification of a *member variable* by creating a *private variable* . This method determines the behavior when assigning a value to those variables.
```